test/format/wts.c

```diff
@@ -76,7 +76,7 @@ compressor(uint32_t compress_flag)
  * encryptor --
  *     Configure encryption.
  */
-static const char *
+const char *
```

Comment:
Currently re-using a function inside wts, so we can add encryption config. Another option would be to re-use the **create_database** function. This is necessary as different databases needs the encryption to be the same, so import can work correctly.